### PR TITLE
Add section in shared/README about React Devtools

### PR DESCRIPTION
**Pull request by akalin-keybase**

### CODE REVIEW

**Comment by keybase-ci-visdiff** [comment]
The commits 58904ac90718...3d15d7e478dc introduce visual changes on linux. <details><summary>:mag_right: 1 changed</summary>

 * Changed: **Profile-Your_Profile_Pending_Proof_Menu_DNS** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1501577789606-58904ac90718-3d15d7e478dc-linux/Profile-Your_Profile_Pending_Proof_Menu_DNS-58904ac90718.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1501577789606-58904ac90718-3d15d7e478dc-linux/Profile-Your_Profile_Pending_Proof_Menu_DNS-3d15d7e478dc.png)  
   Profile-Your_Profile_Pending_Proof_Menu_DNS comparison
</summary>

File: shared/README.md [hunk 1]
```diff
@@ -74,7 +74,25 @@ yarn run rn-gobuild-android
...
 
+### Debugging with React Developer Tools extension
 
+1) Install the [React Developer Tools](https://chrome.google.com/webstore/detail/react-developer-tools/fmkadmapgofadopljbjfkapdkoienihi) in your regular Chrome browser.
+2) Run a yarn command (e.g. `yarn run start`) with the
+`KEYBASE_LOCAL_DEBUG` and `KEYBASE_DEV_TOOL_EXTENSIONS` environment
+variables set appropriately. An example for macOS would be
+
+```
+env KEYBASE_SHOW_DEVTOOLS=1 KEYBASE_LOCAL_DEBUG=1 KEYBASE_DEV_TOOL_EXTENSIONS="$HOME/Library/Application Support/Google/Chrome/Default/Extensions/fmkadmapgofadopljbjfkapdkoienihi/2.5.0_0" yarn run start
```
Comment (chrisnojima) [inline shared/README.md]: Couple of things

1.  I wouldn't suggest inlining this before the command, i'd just suggest settings the environment variable
1. You don't need the actual version number in there, the script itself looks for the version , which also means you can remove the comment below about needing to update it when it changes

Comment (akalin-keybase) [inline shared/README.md]: Done, and updated re. findings from chat.

**Comment by akalin-keybase** [comment]
PTAL!

**Pull request merged**